src/app/external-cluster-wizard/steps/summary/template.html

```diff
@@ -0,0 +1,83 @@
+<!--
+Copyright 2020 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<div class="km-wizard-summary summary-with-number"
+     fxLayout="row wrap"
+     fxLayout.md="column"
+     fxLayout.sm="column"
+     fxLayout.xs="column">
+  <div fxFlex="20%"
+       fxFlex.md="100%"
+       fxFlex.sm="100%"
+       fxFlex.xs="100%"
+       fxLayout="column"
+       class="step">
+    <div class="header counter-1">{{ step.Provider }}</div>
+    <div fxLayout="row wrap">
+      <km-property>
+        <div key>Provider</div>
+        <div value><i class="km-provider-logo km-provider-logo-{{ provider }}"></i></div>
+      </km-property>
+    </div>
+  </div>
+
+  <div fxFlex="20%"
+       fxFlex.md="100%"
+       fxFlex.sm="100%"
+       fxFlex.xs="100%"
+       fxLayout="column"
+       class="step">
+    <div class="header counter-2">{{ step.Settings }}</div>
+
+    <div>
+      <div fxFlex
+           class="container-spacing">
+        <div class="section-header">Credentials</div>
+        <km-property>
+          <div key>Preset</div>
+          <div value>
+            <span>{{ preset ?? 'No preset selected' }}</span>
+          </div>
+        </km-property>
+
+        <km-property *ngIf="cluster.cloud.eks?.region">
+          <div key>Region</div>
+          <div value>
+            <span>{{ cluster.cloud.eks.region }}</span>
+          </div>
+        </km-property>
+      </div>
+    </div>
+  </div>
+
+  <div fxFlex="60%"
+       fxFlex.md="100%"
+       fxFlex.sm="100%"
+       fxFlex.xs="100%"
+       fxLayout="column"
+       class="step">
+    <div class="header counter-3">{{ step.ExternalClusterDetails }}</div>
+
+    <km-external-cluster-details-eks *ngIf="cluster.spec.eksclusterSpec"
```

Comment:
I dont why did you these invoke individually here and than form each `individual` component `km-external-cluster-details` which PROJECTS data into `km-external-cluster-details` into

- <ng-content select="[leftColumn]"></ng-content>
- <ng-content select="[rightColumn]"></ng-content>

reasons?

IMO, don't add complexity in this component what you can do 
pass `cluster` input from `src/app/external-cluster-wizard/steps/summary/template.html` which are already doing for ALL individual and *ngIf inside this `ExternalClusterDetailsComponent` currently (this will be renamed).

wdyt?

can you check normal wizard summary component.

Reply:
After discussion we have decided to keep current implementation,

Reply:
Only the spec for currently selected provider is available at the time of the summary page, all others are null.

Reply:
That's correct but I think it would be more clear if we added the check on selected provider. Wdyt?